tests/BrowserTests.php

```diff
@@ -426,6 +426,4 @@ protected static function catchVarDumperOutput(callable $callback): array
         // a null value is added to the beginning
         return \array_values(\array_filter($output));
     }
-
-    abstract protected function browser(): Browser;
```

Comment:
I was a little confused ensuring the proper browser was being used in all the tests. Could we keep this and have the end tests add the trait they need?